dags/test_nextflow_operator.py

```diff
@@ -37,14 +37,15 @@
               but please use a location under this directory.
            """
            )
-       }
+       },
+       render_template_as_native_obj=True
     ) as dag:
         
         NextflowOperator(
             task_id='test_nextflow_operator',
             name="test_nextflow_operator", # prefix for the pod name
             k8s_context = K8sContext.ETL,
-            arguments =  dag.params["nextflow_command"],
+            arguments = "{{ params.nextflow_command }}",
```

Comment:
PEP 8: E251 unexpected spaces around keyword / parameter equals

Reply:
Ok! Actually, I did not apply any style enforcement tool in my initial commit for `dags/test_nextflow_operator.py` and `dags/lib/operators/nextflow.py`, so there are probably other style inconsistencies. I fixed the one you reported here and a bunch of others I found by running flake8. These fixes are all in the second commit.

I tolerated 3 "line too long" warnings in `dags/lib/operators/nextflow.py`. These were caused by long variable names. I attempted to fix them but preferred the initial version. If you don't tolerate long lines, please let me know.

Reply:
Je pense que les lines too long étaient pour 80 caractères si je me fie à ce qui a été modifié? On peut tolérer 120. 80 c'est court je trouve.